File: look-and-say/look-and-say-java/src/main/java/pl/frognar/_03/Sequence.java
package pl.frognar._03;

public class Sequence {
    public static String lookAndSay(String input) {
        String result = "";
        if (input.length() == 2) {
            if (input.charAt(0) == input.charAt(1)) {
                int count = 1;
                count ++;
                result += "%s%s".formatted(count, input.charAt(0));
            } else {
                result += "1%s".formatted(input.charAt(0));
                result += "1%s".formatted(input.charAt(1));
            }
        } else {
            result += "1%s".formatted(input.charAt(0));
        }

        return result;
    }
}

File: look-and-say/look-and-say-py/01/sequence.py
def look_and_say(input_value: str) -> str:
    result = ""
    if len(input_value) == 2:
        if input_value[0] == input_value[1]:
            count = 2
            result += f"{count}{input_value[0]}"
            return result
        count = 1
        result += f"{count}{input_value[0]}"
        result += f"1{input_value[1]}"
        return result
    result += f"1{input_value}"
    return result
